pymc3/sampling.py

```diff
@@ -695,11 +695,15 @@ def init_nuts(init='auto', njobs=1, n_init=500000, model=None,
         Initialization method to use.
 
         * auto : Choose a default initialization method automatically.
-          Currently, this is `'advi+adapt_diag'`, but this can change in
+          Currently, this is `'unif+adapt_diag'`, but this can change in
```

Comment:
Is uniform random really a good choice? Also, it might be better to sample from the prior, but with the failed case sample from a random normal.